src/cdk-experimental/listbox/listbox.ts

```diff
@@ -89,6 +89,9 @@ export class CdkListbox<V> {
   /** Whether the listbox is disabled. */
   disabled = input(false, {transform: booleanAttribute});
 
+  /** Whether the listbox is readonly. */
+  readonly = input(false, {transform: booleanAttribute});
```

Comment:
Do we also need to bind an `aria-` attr?

Reply:
Whoops, yes we absolutely do